pkg/scheduling/requirement.go

```diff
@@ -48,96 +51,170 @@ func NewRequirement(key string, operator v1.NodeSelectorOperator, values ...stri
 	if operator == v1.NodeSelectorOpIn || operator == v1.NodeSelectorOpNotIn {
 		r.values.Insert(values...)
 	}
+	if operator == v1.NodeSelectorOpGt {
+		value, _ := strconv.Atoi(values[0]) // prevalidated
```

Comment:
I don't see where this is validated.  I updated my provisioner with a bad Lt/Gt requirement and didn't get any validation errors.  The Lt/Lt value must be a) a single value and b) an integer.

Reply:
See the latest commit